client/src/pages/Newsfeed.js

```diff
@@ -7,12 +7,23 @@ import StefanNormal from '../images/FakePostImages/StefanNormal.svg';
 import SimonNormal from '../images/FakePostImages/SimonNormal.svg';
 import MolnarNormal from '../images/FakePostImages/MolnarNormal.svg';
 import styled from 'styled-components/macro';
+import { useState } from 'react';
 
 export default function Newsfeed({ open }) {
 
+    const [isVisible, setVisible] = useState(false);
+    function callbackFunction(childData) {
+        setVisible(!isVisible);
+    }
+
+    function toggleVisible() {
```

Comment:
this function does the same thing as the function above. you only need one of them